tests/test_init.py

```diff
@@ -129,26 +139,32 @@ def test_diff_output_for_diff_seed(self, init, sgntr, seed, tol):
         p1 = init(**s)
         s = {**s, 'seed': seed + 1}
         p2 = init(**s)
-        assert not np.allclose(p1, p2, atol=tol)
+
+        if p1.shape != (0,):
+            assert not np.allclose(p1, p2, atol=tol)
 
     @pytest.mark.parametrize("init, sgntr", INIT_KWARGS)
     def test_interval(self, init, sgntr, seed, tol):
         """Test that sampled parameters lie in correct interval."""
-        s = {**sgntr, 'seed': seed}
 
-        # Case A: Uniformly distributed parameters
-        if 'low' in s.keys() and 'high' in s.keys():
-            s['low'] = 1
-            s['high'] = 1
+        # exclude case of empty parameter list
+        if not init(**sgntr).shape == (0, ):
+
+            s = {**sgntr, 'seed': seed}
+
+            # Case A: Uniformly distributed parameters
```

Comment:
It could be nice to have the uniform and the normal cases separately. In such a case then one could see which fails exactly. I guess in tests it's fine to not worry about code duplication as long as it makes the test cases more explicit upon failing.

Reply:
Good point for future init test refactor!